components/centraldashboard/public/assets/i18n/languages.json

```diff
@@ -295,6 +296,7 @@
             "manageFilersView.duplicateFiler": "Le chemin de partage a déjà été sélectionné",
             "manageFilersView.missingDeleteError": "L'utilisateur n'a pas le chemin de partage {filerShare}",
             "manageFilersView.removeErrorError": "Échec de la suppression de l'erreur",
+            "manageFilersView.lblLatestError": "Dernière erreur",
```

Comment:
French is singular. English is plural. Should probably be "Dernieres erreurs" i think?

Reply:
Oh yes, dom-repeat